qa-tests-backend/scripts/run-part-1.sh

```diff
@@ -84,27 +84,15 @@ test_part_1() {
     rm -f FAIL
     local test_target
 
-    if is_openshift_CI_rehearse_PR; then
-        info "On an openshift rehearse PR, running BAT tests only..."
-        test_target="bat-test"
```

Comment:
I restored it, not a big deal. Maybe it matters because BAT tests run completes in just a bit under 3h, but full test runs a bit more than 3h.

My greater concern was about the need of `QA_TEST_TARGET` variable and that in some contexts we got BAT tests despite we needed all.